android/java/res/layout/brave_leo_clear_history_dialog.xml

```diff
@@ -4,8 +4,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
 
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@deeppandya we need `LinearLayout` in that case as we need to apply `AlertDialogContent` styles to the dialog.